donut/modules/editor/templates/created_list.html

```diff
@@ -1,9 +1,11 @@
 {% extends "layout.html" %}
-{% block page %}
-<head>
-<script src="https://cdn.rawgit.com/showdownjs/showdown/1.0.1/dist/showdown.min.js"></script>
-<script src="{{ url_for('uploads.static', filename='script.js', _external = True)}}" type="text/javascript"></script>
-</head>
+{% block scripts %}
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Including jQuery shouldn't be necessary since it's already included on every page